package.json

```diff
@@ -269,7 +260,7 @@
     "test": "jest src/",
     "test:ci": "jest src/ --coverage",
     "test:watch": "jest --watch src/",
-    "lint": "eslint \"{src,playwright}/**/*.@(tsx|ts|jsx|js)\" --fix",
+    "lint": "cross-env NODE_OPTIONS=--max-old-space-size=8192 eslint \"{src,playwright}/**/*.@(tsx|ts|jsx|js)\" --fix",
```

Comment:
I had to add this because **sometimes** ESLint run out of memory on my machine. We already use more memory for other scripts in dashboard as well.